src/replicache.ts

```diff
@@ -78,6 +78,12 @@ export interface ReplicacheOptions {
    */
   name?: string;
 
+  /**
+   * The schema version of the data understood by this application. This allows
```

Comment:
not clear to me that the same schema version is used for pull and push. we can assert this for starters i guess and see if anyone asks us to provide separate push_ and pull_schema_versions.